parcels/kernel.py

```diff
@@ -353,10 +353,12 @@ def remove_deleted(pset):
             for p in error_particles:
                 if p.state == ErrorCode.Repeat:
                     p.state = ErrorCode.Success
-                else:
+                elif p.state in recovery_map:
                     recovery_kernel = recovery_map[p.state]
                     p.state = ErrorCode.Success
                     recovery_kernel(p, self.fieldset, p.time)
+                else:
```

Comment:
Add here in comments that this is a `HOTFIX for #750`?

Reply:
agreed